lib/cn/cn_tree.c

```diff
@@ -1356,71 +1359,103 @@ cn_comp_update_kvcompact(struct cn_compaction_work *work, struct kvset *new_kvse
     }
 }
 
-/**
- * cn_comp_update_spill() - update tree after spill operation
- * See section comment for more info.
- */
+static bool
+cn_node_spill_wait(struct cn_compaction_work *w, struct cn_tree_node *tn)
```

Comment:
Maybe we can avoid passing 'tn' here as both w->cw_node and 'tn' are the same, right?